src/OrbitGl/include/OrbitGl/CaptureViewElement.h

```diff
@@ -164,13 +164,24 @@ class CaptureViewElement : public Pickable, public AccessibleInterfaceProvider {
   [[nodiscard]] virtual EventResult OnMouseEnter();
   [[nodiscard]] virtual EventResult OnMouseLeave();
 
+  [[nodiscard]] virtual bool RestrictDrawingToBody() const { return false; }
+
+  [[nodiscard]] uint32_t GetUid() const { return uid_; }
+
  private:
   bool is_mouse_over_ = false;
+  uint32_t uid_;
+
+  std::string previous_batcher_render_group_name_{BatchRenderGroupId::kGlobalGroup};
+  std::string previous_text_render_group_name_{BatchRenderGroupId::kGlobalGroup};
 
   float width_ = 0.;
   Vec2 pos_ = Vec2(0, 0);
   CaptureViewElement* parent_;
 
+  void PreRender(PrimitiveAssembler& primitive_assembler, TextRenderer& text_renderer);
+  void PostRender(PrimitiveAssembler& primitive_assembler, TextRenderer& text_renderer);
+
```

Comment:
Where is this used? Should it be in the next PR?

Reply:
Ah, yes, this should be in the follow-up PR. I'll remove it.